README.md

```diff
@@ -181,6 +181,8 @@ The questionnaire directive accepts the following options:
   (select N questions randomly on each load)
 * `category`: exercise category
 * `status`: exercise status (default "unlisted"). See available [statuses](#list-of-exercise-statuses).
+* `allow-assistant-viewing`: Allows assistants to view the submissions of the students. Can be set to true or false. Overrides any options set in the conf.py or config.yaml files.
+* `allow-assistant-grading`: Allows assistants to grade the submissions of the students.Can be set to true or false. Overrides any options set in the conf.py or config.yaml files.
```

Comment:
One space is missing after one period. You don't need to write everything on a single line when the line becomes too long.